.travis.yml

```diff
@@ -89,13 +89,17 @@ install:
 script:
   - cd ${TRAVIS_BUILD_DIR}
   - if [[ $TRAVIS_OS_NAME == 'osx' ]]; then ulimit -S -n 2048; fi
+  - if [[ ${TRAVIS_PULL_REQUEST} == "true" && \
```

Comment:
I wanted to check my code as best as I could before we merge this to master to make sure it works. So yeah this will be removed before the merge